main.py

```diff
@@ -65,27 +68,50 @@
             patchSize = hf['patchSize'][:]
 
     else: # perform patching
-        dAllPatches = np.zeros((0, patchSize[0], patchSize[1]))
-        dAllLabels = np.zeros(0)
-        dAllPats = np.zeros((0, 1))
-        lDatasets = cfg['selectedDatabase']['dataref'] + cfg['selectedDatabase']['dataart']
-        iLabels = cfg['selectedDatabase']['labelref'] + cfg['selectedDatabase']['labelart']
-        for ipat, pat in enumerate(dbinfo.lPats):
-            if os.path.exists(dbinfo.sPathIn + os.sep + pat + os.sep + dbinfo.sSubDirs[1]):
-                for iseq, seq in enumerate(lDatasets):
-                    # patches and labels of reference/artifact
-                    tmpPatches, tmpLabels  = datapre.fPreprocessData(os.path.join(dbinfo.sPathIn, pat, dbinfo.sSubDirs[1], seq), cfg['patchSize'], cfg['patchOverlap'], 1, cfg['sLabeling'])
-                    dAllPatches = np.concatenate((dAllPatches, tmpPatches), axis=0)
-                    dAllLabels = np.concatenate((dAllLabels, iLabels[iseq]*tmpLabels), axis=0)
-                    dAllPats = np.concatenate((dAllPats, ipat*np.ones((tmpLabels.shape[0],1), dtype=np.int)), axis=0)
+        X_train = []
```

Comment:
This looks good.
@TokenJan @ZiweiWU713 @SebastianMilde Can you confirm that the previous version is still running with these modifications?